src/main/java/com/iexec/sms/tee/config/TeeWorkerInternalConfiguration.java

```diff
@@ -0,0 +1,60 @@
+package com.iexec.sms.tee.config;
+
+import com.iexec.sms.api.config.TeeAppProperties;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.util.unit.DataSize;
+import org.springframework.validation.annotation.Validated;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Positive;
+
+@Configuration
+@Validated
+public class TeeWorkerInternalConfiguration {
+    @Bean
+    TeeAppProperties preComputeConfiguration(
```

Comment:
`preComputePropertiesBean`?

Reply:
We don't ever have a bean with `bean` suffix. I'm not sure this would help anyone to add this suffix there.